### PR TITLE
Adding Apache 2.0 License

### PR DESCRIPTION
**Pull request by kkeene44**
TYPE: text only

KEYWORDS: license, apache-2.0,

SOURCE: internal

DESCRIPTION OF CHANGES: This license is for open source code. It is suitable for research, academia, and commercial use of WRF. It allows the source code to be openly available, but does not endorse its usage. All contributors are also bound by this license.

LIST OF MODIFIED FILES: 
A    LICENSE

TESTS CONDUCTED: none

### CODE REVIEW

**Comment by davegill** [comment]
@llpcarson @duda @kkeene44 
Folks,
Are you OK with this recognizable license for WRF and WPS?

File: LICENSE [hunk 1]
```diff
@@ -0,0 +1,201 @@
...
+      License. However, in accepting such obligations, You may act only
+      on Your own behalf and on Your sole responsibility, not on behalf
+      of any other Contributor, and only if You agree to indemnify,
+      defend, and hold each Contributor harmless for any liability
+      incurred by, or claims asserted against, such Contributor by reason
+      of your accepting any such warranty or additional liability.
+
+   END OF TERMS AND CONDITIONS
+
+   APPENDIX: How to apply the Apache License to your work.
```
Comment (davegill) [inline LICENSE]: @kkeene44 @duda @llpcarson 
Do we need to do something here?

**Comment by llpcarson** [comment]
Kelly - We use a much simpler reference to the Apache 2.0 license for some of our other work - attached here.  Might be more straightforward?

Also, generally, with the Apache license, one would also include a Copyright clause (which is often included in every file).  With wide-ranging contributors, I'm not sure if you would use a simple UCAR copyright?
[license.txt](https://github.com/wrf-model/WRF/files/2650030/license.txt)

I don't have any objection to the Apache license in general, but copyright issues might be tricky going from public domain to licensed open source!

File: LICENSE [hunk 2]
```diff
@@ -0,0 +1,201 @@
...
+      To apply the Apache License to your work, attach the following
+      boilerplate notice, with the fields enclosed by brackets "[]"
+      replaced with your own identifying information. (Don't include
+      the brackets!)  The text should be enclosed in the appropriate
+      comment syntax for the file format. We also recommend that a
+      file or class name and description of purpose be included on the
+      same "printed page" as the copyright notice for easier
+      identification within third-party archives.
+
+   Copyright [yyyy] [name of copyright owner]
```
Comment (dmey) [inline LICENSE]: This should be filled in

Comment (llpcarson) [reply LICENSE]: Agree completely with the copyright info - my example was from a different project (not-WRF), so had the NOAA/UCAR/CIRES/etc....  whatever is appropriate for WRF should be included!

**Comment by davegill** [comment]
@dmey 
Thanks for the correction on the intended person for the comment.

**Comment by davegill** [comment]
@dmey 
Thanks for the correction on the intended person for the comment.

**Comment by powersjg** [comment]
Kelly:
Just a request for an edit to the description of changes.
Delete: "It maintains our copyright."  The reason is that we don't have a copyright on WRF;
we declined to pursue copyrighting early on. As open source code, WRF is free for all to copy and redistribute.

Thanks,
Jordan

**Comment by kkeene44** [comment]
@powersjg Thanks for pointing that out. I made the modification.

@davegill @weiwangncar  @llpcarson 
I don't think we want something that requires including a copyright clause in every file - especially since, as was pointed out above, we don't have a copyright on WRF. Perhaps something more like the MIT license would have more relevancy to our code. 

@mgduda 
Based on what you guys do for MPAS, do you have any suggestions?

**Comment by davegill** [comment]
@kkeene44 
Several people have mentioned a simpler license, which I am inclined to lean towards.

**Pull request closed without merging**